src/fondue/Fondue.js

```diff
@@ -586,4 +586,130 @@ export default class Fondue {
 
 		return charset;
 	}
+
+	// Return characters per feature
+	// Only GSUB for now, TODO: GPOS
+	get featureChars() {
```

Comment:
@RoelN I'm familiar enough with the domain to properly review this.
In general (as you know by now :-P) I'm not a fan of loops and mutation like this code does, but it is structured quite well it seems so it's not really a problem.

I do wonder about the performance impact of this code. It might be nice to start adding some perf tests to WF engine.